app/views/about/index.html.erb

```diff
@@ -1,5 +1,125 @@
-<div class="dm-about-page">
-  <div class="dm-system-message">
-    <%= render partial: "shared/messages", locals: {small_text: false} %>
+<div class="dm-about-page grid-container">
+  <div class="dm-system-message margin-top-2">
+    <%= render partial: "shared/messages", locals: { small_text: true } %>
   </div>
+
+  <section class="about-us-info-section margin-y-128" role="region" aria-label="About us info section">
+    <div class="grid-row grid-gap-3">
+      <div class="grid-col-7">
+        <h1 class="usa-prose-h1 margin-bottom-4 margin-top-0">About us</h1>
+        <p class="usa-prose-body">
+          The Diffusion Marketplace is a discovery and collaboration tool that curates promising clinical, operational, and strategic practices from VA. We are managed by the
+          <a href="https://www.va.gov/INNOVATIONECOSYSTEM/views/who-we-are/diffusion-of-excellence.html" class="dm-primary-link" target="_blank">Diffusion of Excellence</a>, an organization within
```

Comment:
I'm not sure. In the homepage branch, I created a new `dm-primary-link` that we can use if we need the bold stuff.

Reply:
if it's just this page I would use the `usa-link text-bold`. We'll wait and see what Becca says. I also asked if we should make this a `usa-link usa-link--external text-bold` link since it is technically outside of Diffusion Marketplace.

Reply:
Okay, I refactored them and removed the `dm-primary-link` style